src/content/docs/en/guides/integrations-guide/vercel.mdx

```diff
@@ -10,9 +10,9 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
 import Since from '~/components/Since.astro';
 import ReadMore from '~/components/ReadMore.astro'
 
-This adapter allows Astro to deploy your [`hybrid` or `server` rendered site](/en/basics/rendering-modes/#on-demand-rendered) to [Vercel](https://www.vercel.com/).
+This adapter allows Astro to deploy your [`static`, `hybrid` or `server` rendered site](/en/basics/rendering-modes/#on-demand-rendered) to [Vercel](https://www.vercel.com/) and configure additional services such as [Vercel Web Analytics](https://vercel.com/docs/analytics).
 
-If you're using Astro as a [static site builder](/en/basics/rendering-modes/#pre-rendered), you don't need an adapter.
+If you're using Astro as a [static site builder](/en/basics/rendering-modes/#pre-rendered) and do not need to set up services such as [Vercel Web Analytics](https://vercel.com/docs/analytics) and [Vercel Image Optimization](https://vercel.com/docs/image-optimization) or use a custom configuration, you don't need an adapter.
```

Comment:
```suggestion
If you're using Astro as a [`static` site builder](/en/basics/rendering-modes/#pre-rendered), you only this adapter if you are using additional Vercel services (e.g. [Vercel Web Analytics](https://vercel.com/docs/analytics), [Vercel Image Optimization](https://vercel.com/docs/image-optimization)). Otherwise, you do not need an adapter to deploy your `static` site.
```
I think keeping all the static stuff together makes sense!